sunkit_instruments/goes_xrs/goes_xrs.py

```diff
@@ -168,1134 +83,6 @@ def get_goes_event_list(timerange, goes_class_filter=None):
     return goes_event_list
 
 
-def calculate_temperature_em(
```

Comment:
Like with the other functions in this module, can we keep this but have it raise a `NotImplementedError`/`DeprecatedError` to tell people where to go to use the new version of these functions?